src/main/java/com/google/sps/data/Election.java

```diff
@@ -92,11 +92,10 @@ public static Election fromElectionQuery(JSONObject electionQueryData) throws JS
   }
 
   /**
-   * Creates a new Election object by extracting the properties from an Election object and from the
```

Comment:
"Creates" is actually more accurate because toBuilder actually creates a new object. 

https://github.com/google/auto/blob/master/value/userguide/builders-howto.md#-initialize-a-builder-to-the-same-property-values-as-an-existing-value-instance
```
... but it would be convenient if they could easily get a Builder instance representing the same 
property values, which they could then modify and **use to create a new value instance.**
```